cyclonedx/model/__init__.py

```diff
@@ -1372,6 +1239,9 @@ def __repr__(self) -> str:
         return f'<Copyright text={self.text}>'
 
 
+# Importing here to avoid a circular import
+from .tool import Tool  # pylint: disable=wrong-import-position # noqa: E402
+
 ThisTool = Tool(
```

Comment:
I am planning reworks on `ThisTool`, and also added something that creates a Component to represent this very python library.
see #649

If I were you, I would simply keep `ThisTool` things you have them now, and let me take care it, after your PR is merged eventually ;-)

Reply:
Fantastic! I tried doing a "Component" ThisTool, and that broke a lot of things, and I wasn't sure what the best fix was, so that's great to hear. :) I'm going to try to see what additional test I need and try to finish up this PR today.